ConsentViewController/Classes/SourcePointClient/MessageResponse.swift

```diff
@@ -88,6 +88,7 @@ struct MessageMetaData: Decodable, Equatable {
     let categoryId: MessageCategory
     let subCategoryId: MessageSubCategory
     let messageId: Int
+    let prtnUUID: String
```

Comment:
Same here. We can give `prtnUUID` a friendlier name and override `CodingKey` enum to match the payload name.